### PR TITLE
Use shx for the build-site script, clean before run

### PR DESCRIPTION
**Pull request by MoonE**
The build-site script does not use shx to execute its copy / move / mkdir commands and thus cannot be run in e. g. windows cmd.
The script also fails to copy the examples / apidoc when it is run a second time because folders cannot be move when there they already exist in the destination.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,12 +23,12 @@
...
+    "build-site": "shx rm -rf build/site && npm run build-examples && npm run apidoc && shx mkdir -p build/site && shx cp site/index.html build/site/ && shx mv build/apidoc build/examples build/site/",
     "copy-css": "shx cp src/ol/ol.css build/ol/ol.css",
     "generate-types": "npx --package=typescript@3.8.3 -y -- tsc --project config/tsconfig-build.json --declaration --declarationMap --emitDeclarationOnly --outdir build/ol",
     "transpile": "shx rm -rf build/ol && shx mkdir -p build/ol && shx cp -rf src/ol build/ol/src && node tasks/serialize-workers.cjs && npx --package=typescript@4.3.5 -y -- tsc --project config/tsconfig-build.json",
     "typecheck": "npx --package=typescript@4.3.5 -y -- tsc --pretty",
-    "apidoc-debug": "shx rm -rf build/apidoc && node --inspect-brk=9229 ./node_modules/jsdoc/jsdoc.js -R config/jsdoc/api/index.md -c config/jsdoc/api/conf.json -P package.json -d build/apidoc",
+    "apidoc-debug": "shx rm -rf build/apidoc && node --inspect-brk=9229 ./node_modules/.bin/jsdoc -R config/jsdoc/api/index.md -c config/jsdoc/api/conf.json -P package.json -d build/apidoc",
```
Comment (ahocevar) [inline package.json:31]: Is this change necessary? I seem to remember that there were problems with dot-directories in Windows under some circumstances.

Comment (MoonE) [reply package.json]: It's not necessary.
As I understand it the .bin directory is where links to all the executable files are added to make it easier to run them.
Thought it make sense to use it, but I wasn't aware of potential problems.

**Review by ahocevar — APPROVED** [review]
